Examples/Io/Root/src/RootTrackSummaryWriter.cpp

```diff
@@ -485,6 +487,15 @@ ActsExamples::ProcessCode ActsExamples::RootTrackSummaryWriter::writeT(
       m_cov_eT_eQOP.push_back(getCov(5, 4));
       m_cov_eT_eT.push_back(getCov(5, 5));
     }
+
+    if (tracks.hasColumn(Acts::hashString("Gx2fnUpdateColumn"))) {
```

Comment:
You could also think about putting the exact literal in a `constexpr std::string_view` somewhere, so it can be changed in one place consistently.
I have done this for the GSF here: https://github.com/acts-project/acts/blob/1c81427c73376a48608daac71373b5d07cb18f50/Core/include/Acts/TrackFitting/GsfOptions.hpp#L38